src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -277,6 +292,7 @@ class FormPage extends React.Component {
           onChange={this.onChange}
           onSubmit={this.onSubmit}
         >
+          {pageContentBeforeButtons}
```

Comment:
Does this need to fallback to `null`? In my head, I keep thinking this could render "undefined" as text.

Reply:
oh good point, I'll add a FormPage test. As for the null/undefined, this seems to be okay. I think the problem is when you have a "0"

Reply:
added test